ironfish/src/wallet/wallet.ts

```diff
@@ -659,6 +659,11 @@ export class Wallet {
       throw new Error('Cannot start a scan if accounts are not loaded')
     }
 
+    if (!this.isStarted) {
```

Comment:
This will break one of the uses of `ironfish wallet:rescan`:

If you run `ironfish wallet:rescan -f` without a node running it will start a rescan even though the wallet isn't started

We could check the config value here instead of `isStarted` 🤔

Reply:
I see different behavior: 

<img width="1387" alt="image" src="https://github.com/iron-fish/ironfish/assets/13268167/3924d628-8fbf-430d-ad89-0c7f698991c0">

This is what the node says: 

<img width="847" alt="image" src="https://github.com/iron-fish/ironfish/assets/13268167/d04ffceb-210c-4944-9b09-0de0fcf01a81">

Reply:
I only see 1 place in the code where `isStarted` is set: 

<img width="1440" alt="image" src="https://github.com/iron-fish/ironfish/assets/13268167/29a4bc1f-e4e9-4158-912b-3975d89d5e74">

Reply:
The rescan case I'm talking about is a bit different:

- `enableWallet` is set to `true`
- node is _not_ running
- run `ironfish wallet:rescan -f`

The wallet should scan in this case, but I'm not sure it will because `isStarted` is false